ci/authn-oidc/phantomjs/keycloak_login.js

```diff
@@ -48,14 +52,13 @@ steps = [
     },
 
     function(){
-		     console.log("Wait keycloak to login user");
-         var fs = require('fs');
+		     log("Wait for keycloak to login user");
```

Comment:
Can you please fix the indenting here? (And everywhere else in the file that it's slightly off).